recipes/pgpy/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "pgpy" %}
+{% set version = "0.5.4" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/PGPy-{{ version }}.tar.gz
+  sha256: bdd3da1e006fc8e81cc02232969924d6e8c98a4af1621a925d99bba09164183b
+
+build:
+  number: 0
+  skip: true   # [py==30 or py==31 or py==32 or py==33]
```

Unless you personally need support for Python 2.7, better to make this `noarch: python` since it's pure Python:
```suggestion
  noarch: python
```

Then you can remove the enum/singledispatch dependencies below and just depend on e.g. `python >=3.5`

---

Thanks @dopplershift, I've made those changes.